tests/external_botocore/test_bedrock_chat_completion.py

```diff
@@ -143,6 +147,15 @@ def expected_events_no_content(expected_events):
     return events
 
 
+@pytest.fixture(scope="module")
+def expected_events_with_token_count(expected_events):
+    events = copy.deepcopy(expected_events)
+    for event in events:
+        if event[0]["type"] != "LlmChatCompletionSummary":
```

Comment:
```suggestion
        if event[0]["type"] == "LlmChatCompletionMessage":
```

Reply:
This was kept with the negative check because the same function is used for embeddings.